src/feed/api.ts

```diff
@@ -0,0 +1,25 @@
+import { Bee, Utils } from '@ethersphere/bee-js'
+import { Data } from '@ethersphere/bee-js/dist/src/types'
+import { bmtHashString, extractChunkData } from '../account/utils'
+import { getId } from './handler'
+import { bytesToHex } from '../utils/hex'
+import { keccak256Hash } from '../account/encryption'
+import { lookup } from './lookup/linear'
+import { Epoch } from './lookup/epoch'
+import Long from 'long'
+import { NoClue } from './lookup/lookup'
+
+const downloadTimeout = 1000
+
+export async function getFeedData(bee: Bee, topic: string, address: string): Promise<Data> {
+  const addressBytes = Utils.makeEthAddress(address)
+  const topicHash = bmtHashString(topic)
+  const chunk = await lookup(Long.fromNumber(0), NoClue, async (epoch: Epoch, time: Long): Promise<Data> => {
+    const tempId = getId(topicHash, time.toNumber(), epoch.level)
```

Comment:
I just don't get you transfer Long into `number` it loses its precision then why are you using `long`?

Reply:
When comparing golang long for the first time, it seemed to me that the accuracy might be lost. It might be worth exploring a little more, but taking into account all the other methods (some of which are not yet in our codebase) and their interrelationships.

Reply:
I think if it is clear what is exactly stored in the bytearray then clears out the correct handling of the data as well. Now, it seems it is just a timestamp + level